ga4gh/datamodel/reads.py

```diff
@@ -251,6 +262,14 @@ def getSamFilePath(self):
         """
         return self._samFilePath
 
+    def getNumAlignedReads(self):
```

Comment:
We should really have some docstring comment here for these methods. It is a bad idea having duplicated documentation in both subclasses, so I think the best thing to do is to define this methods in the superclass, along with appropriate docstring comments, and for them to raise a `NotImplementedError`. See `AbstractVariantSet.getNumVariants` for an example.

This is a good way of being explicit about the superclasses interface too I think.